app/services/checksum_validator.rb

```diff
@@ -43,19 +43,18 @@ def moab_on_storage_absent?
     !File.exist?(object_dir) || latest_moab_storage_object_version.nil?
   end
 
-  def update_complete_moab_status(status)
-    status_handler.update_status(status)
-  end
+  delegate :update_complete_moab_status, to: :status_handler
 
   # Moab on storage and complete moab versions match?
   def versions_match?
     moab_on_storage.current_version_id == complete_moab.version
   end
 
   def validate_versions
-    # set_status_as_seen_on_disk will update results and complete_moab
-    status_handler.set_status_as_seen_on_disk(found_expected_version: versions_match?, moab_on_storage_validator: moab_on_storage_validator,
-                                              caller_validates_checksums: true)
+    # validate_moab_on_storage_and_set_status will update results and complete_moab
```

Comment:
is this a method comment?  Ruby practice is outside the block.

Reply:
No, it is a comment about the following line not the entire method.